src/cs/Ssh/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.DevTunnels.Ssh.Test")]
+[assembly: InternalsVisibleTo("Microsoft.DevTunnels.Ssh.Tcp")]

```

Comment:
I don't think the internals need to be visible to the TCP assembly. It doesn't seem to be used, and anyway I'd rather ensure cross-assembly APIs are public.

As for the tests' use of the internal `KeepAliveRequestReceived` I have a separate suggestion about that, so that can be removed also.

Reply:
The TCP assembly was adding the `TaskExtensions.cs` explicitly and I removed that from Test assembly and did the same change here.

Reply:
I'm not a big fan of `InternalsVisibleTo` because it can lead to unintentional use of internal APIs across assemblies, or laziness in not creating proper public APIs.